workos/typing/untyped_literal.py

```diff
@@ -4,7 +4,7 @@
 
 
 class UntypedLiteral(str):
-    def __new__(cls, value: str):
+    def __new__(cls, value: str) -> "UntypedLiteral":
```

Comment:
I'm kind of shocked this isn't just inferred.

Reply:
Same :sad_panda: